src/store/wizardSlice.ts

```diff
@@ -824,7 +824,9 @@ export const wizardSlice = createSlice({
     },
     addPackage: (state, action: PayloadAction<IBPackageWithRepositoryInfo>) => {
       const existingPackageIndex = state.packages.findIndex(
-        (pkg) => pkg.name === action.payload.name,
+        (pkg) =>
```

Comment:
This would allow user to add duplicate packages (by name) if stream was different, wouldn't it? :eyes: 

I don't think we want that and I'd assume the UI is blocking this by disabling correct checkboxes.